### DISCUSSION TITLE
v3.2.0

### DISCUSSION THREAD

**Discussion by InfinityLoop1308** [InfinityLoop1308/PipePipe#194] [Announcements]
Other translations(zh-rCN, zh-rTW, jp) available [here](https://github.com/InfinityLoop1308/PipePipe/wiki/Update-Log#v320)
### New Features:

- Support downloading the whole playlist (beta, only for < 50 items playlists)
- Support show images in comments
- Support navigate to the filtered item when filtering local playlists
- Support sorting local playlists
- Support SponsorBlock (YouTube) [GitHub #38, CodeBerg #39]
- Support fetching the stream size before download (BiliBili)
- Support download video concurrently (BiliBili)
- Support display Staff List for cooperation videos (BiliBili)
- Support showing more stream statistics (BiliBili)

### Bug Fixes:

- Feeds can not be fetched under specific circumstances [CodeBerg #113]
- Resolution settings not working on non-YouTube services [GitHub #88, #125, #158, #189, CodeBerg #19, #50]
- App not working on Samsung OneUI Android 14 [CodeBerg #110, #118, GitHub #176, #178]
- Bullet comments not work as expected (YouTube) [GitHub #118, #111]
- Partitions are enqueued even if a playlist is playing (BiliBili)

### Others:

- Update some desciptions
- Provide translations for zh-cn, zh-tw, and jp

<hr/>

### Which APK should you download?

- arm64-v8a should work on most devices.
- armeabi-v7a if your device is a 10+ year old phone.
- x86_64 for 64-bit Android simulators (on a PC).
- x86 for 32-bit simulators.

<hr /><em>This discussion was created from the release <a href='https://github.com/InfinityLoop1308/PipePipe/releases/tag/v3.2.0'>v3.2.0</a>.</em>

**Comment by logicaldeveloper** [comment]
I am using new pipe for YouTube with sponsor block and i was using pipepipe for bilibili. Thanks a million ton for enabling sponsorblock. I can now use only one app

**Reply by InfinityLoop1308** [reply]
I'm glad you like it. Feel free to share it with more people.🎉
